dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/category/hibernate/CategoryOptionCombo.hbm.xml

```diff
@@ -15,7 +15,7 @@
     </id>
     &identifiableProperties;
 
-    <property name="name" column="name" access="property" unique="false" not-null="false" type="text" index="in_categoryoptioncombo_name" />
```

Comment:
Is there a reason we are making name not-unique now?

Reply:
it was non-unique before as well. unique is false by default so I have removed it.